### DISCUSSION TITLE
Implementing piotr22's AFSK messaging

### DISCUSSION THREAD

**Discussion by sn0ren** [egzumer/uv-k5-firmware-custom#408] [Ideas]
I love the egzumer firmware - but I also really love the possibilities of the AFSK messenger mod from piotr022. Would it be possible to integrate that into egzumer, or possibly choose between having either spectrum analyser or messenger?

https://github.com/piotr022/UV_K5_playground

**Comment by egzumer** [comment]
https://github.com/joaquimorg/uv-k5-firmware-custom

ENABLE_MESSENGER

**Reply by codingncaffeine** [reply]
When I hit F Key + Menu it takes me to D Prel in the menu.  I'm on the latest version of egzumer.  Is this working at intended?
